docs/developers/guides/accounts/create-account-key.md

```diff
@@ -5,15 +5,15 @@ An account can authorize account keys, which can create messages on its behalf.
 The owner of the account can revoke an account key at any time. To add an account key, you'll need to follow six steps:
 
 1. Set up [Viem](https://viem.sh/) clients and [`@farcaster/hub-web`](https://www.npmjs.com/package/@farcaster/hub-web) account key.
-2. Register an [app fid](/reference/contracts/faq#what-is-an-app-fid-how-do-i-get-one) if your app does not already have one.
+2. Register an [app FID](/reference/contracts/faq#what-is-an-app-fid-how-do-i-get-one) if your app does not already have one.
 3. Create a new account key for the user.
 4. Use your app account to create a [Signed Key Request](/reference/contracts/reference/signed-key-request-validator).
 5. Collect an [`Add`](/reference/contracts/reference/key-gateway#add-signature) signature from the user.
-6. Call the [Key Gateway](https://docs.farcaster.xyz/reference/contracts/reference/key-gateway#addFor) contract to add the key onchain.
+6. Call the [Key Gateway](https://docs.farcaster.xyz/reference/contracts/reference/key-gateway#addFor) contract to add the key on-chain.
```

Comment:
The community has generally settled on `onchain` and `offchain` as individual words, rather than hyphenated.

Reply:
@sds please review the latest changes (onchain and offchain): [655e6d2](https://github.com/farcasterxyz/docs/pull/296/commits/655e6d2f6bf887400fe7372486fd30be9e9ee908)